src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -61,7 +63,7 @@ def __init__(self, ensemble: Ensemble, config: EvaluatorServerConfig):
         self._ensemble: Ensemble = ensemble
 
         self._events: asyncio.Queue[Event] = asyncio.Queue()
-        self._events_to_send: asyncio.Queue[Event | HeartbeatEvent] = asyncio.Queue()
+        self._events_to_send: asyncio.Queue[Event] = asyncio.Queue()
```

Comment:
This does not match now since down the line you expect:
```python
event: Event | _HeartbeatEvent = await asyncio.wait_for(
                    self._events_to_send.get(), timeout=HEARTBEAT_TIMEOUT
                )
```